next.config.js

```diff
@@ -2,6 +2,11 @@ const withHashicorp = require('@hashicorp/platform-nextjs-plugin')
 const path = require('path')
 const redirects = require('./redirects.next.js')
 
+// Set api key for Happy Kit feature flags
+const happyKitKey = process.env.NEXT_PUBLIC_FLAGS_ENV_KEY
+  ? process.env.NEXT_PUBLIC_FLAGS_ENV_KEY
+  : 'flags_pub_development_343442393171755603'
```

Comment:
Just confirming: this is the key for devdot?

Reply:
It is yes.